examples/sample_simplex_main.cpp

```diff
@@ -1,176 +0,0 @@
-// VolEsti
```

Comment:
you want this to be removed ? We should have a sampling from simplex functionality since it is already implemented